app/src/main/java/com/greenfox/tribesoflagopusandroid/MainActivity.java

```diff
@@ -39,29 +49,33 @@ protected void onCreate(Bundle savedInstanceState) {
         editor = preferences.edit();
         checkUsername();
 
-        Button button = (Button) findViewById(R.id.logout);
-        button.setOnClickListener(new View.OnClickListener() {
-            public void onClick(View v) {
-                logout();
-            }
-        });
-
         MainFragment mainFragment = new MainFragment();
         FragmentManager fragmentManager = getSupportFragmentManager();
         FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
-        fragmentTransaction.add(R.id.main_activity_layout, mainFragment);
+        fragmentTransaction.add(R.id.flContent, mainFragment);
```

Comment:
@rekagiday ids should have either `camelCase` or `lower_under_score` style naming but use the same, and meaningful names, what is `flContent`